src/Core/src/Window.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+
+namespace Microsoft.Maui
+{
+	public class Window : IWindow
```

Comment:
I have doubts if this should be on Core or Controls. Why do we need this on Core ?